src/pages/main-site/About.js

```diff
@@ -12,11 +12,15 @@ import Row from "react-bootstrap/Row";
 import Col from "react-bootstrap/Col";
 import Button from "react-bootstrap/Button";
 
+
+
 const About = () => {
   return (
     <div className="landing">
       <Header />
+      <div style={{padding:"20px"}}>
```

Comment:
What's this div doing? If it's to provide spacing to the Container that comes after, we can apply a margin to the container